src/codegen/revert.rs

```diff
@@ -402,14 +448,58 @@ mod tests {
     }
 
     #[test]
-    fn function_selector_expression() {
+    fn default_error_selector_expression() {
+        let ns = Namespace::new(Target::default_polkadot());
         assert_eq!(
             ERROR_SELECTOR,
-            SolidityError::String(Expression::Poison).selector(),
+            SolidityError::String(Expression::Poison).selector(&ns),
         );
         assert_eq!(
             PANIC_SELECTOR,
-            SolidityError::Panic(PanicCode::Generic).selector(),
+            SolidityError::Panic(PanicCode::Generic).selector(&ns),
         );
     }
+
+    /// Tests the error selector calculation to be correct against two examples:
+    /// - `error ERC20InsufficientBalance(address sender, uint256 balance, uint256 needed);`
+    /// - `error Unauthorized();`
+    ///
+    /// Error selector calculation uses the same signature algorithm used for message selectors.
+    /// Which is expected to already have sufficient test coverage.
```

Comment:
```suggestion
    /// Error selector calculation uses the same signature algorithm used for message selectors,
    /// which is expected to already have sufficient test coverage.
```

Reply:
Can you point where the expected tests are (in your comment)?

Reply:
I made this comment to justify as to why this test is not a test for our signature generation algorithm. There are a many different tests for the signature algorithm across the codebase, for example in sema directly, with function overloading, in the ABI encoding, in the integration tests, in the unit tests, and maybe other places I can't immediately think of. How would adding this list improve this comment?

Reply:
You wrote that this is `expected` to have tests elsewhere, but it is not clear for me where they are.